src/components/Button/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Link from 'gatsby-link';
+
+const Button = ({ division, children, link }) => {
+  const location = link.substr(0, 4) == 'http'
```

Comment:
@dylankfernandes You should always use `===`. There are rare edge cases where `==` will validate as true when they shouldn't